backend/src/main/java/com/cgi/retrospecto/backend/poker/controller/dto/CreateStoryDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.cgi.retrospecto.backend.poker.controller.dto;
+
+public class CreateStoryDTO {
```

Comment:
Same as in CreateRoomDTO, this class just has one field, the code can be simpler without this dto.